app/src/main/java/com/bogdan/codeforceswatcher/redux/states/UIState.kt

```diff
@@ -10,18 +9,30 @@ data class UIState(
 
     enum class HomeTab {
```

Comment:
You should set titleId, menuId directly for each enum. In Kotlin it's possible to add variables for each enum value.